agent/src/beerocks/monitor/monitor_stats.cpp

```diff
@@ -457,6 +460,58 @@ bool monitor_stats::add_ap_metrics(ieee1905_1::CmduMessageTx &cmdu_tx, const sMa
     return true;
 }
 
+bool monitor_stats::add_ap_assoc_sta_traffic_stat(ieee1905_1::CmduMessageTx &cmdu_tx,
+                                                  monitor_sta_node &sta_node)
```

Comment:
sta_node appears only on the rhs, so probably it should be `const`
```suggestion
                                                  const monitor_sta_node &sta_node)
```